recipes/by-provider/amazon.js

```diff
@@ -20,5 +20,5 @@ module.exports = (url, opts) =>
 
 module.exports.meta = {
   name: 'Amazon',
-  examples: ['https://www.amazon.com/Images-SI-Uranium-Ore/dp/B000796XXM']
```

Comment:
Unfortunately for some reason the elements we target aren't present on your uranium ore 😅, so I've replaced the example product.